pom.xml

```diff
@@ -865,13 +866,13 @@ SPDX-License-Identifier: MIT
                   <!-- parse the version number from `INSERT INTO metadata (id, config_key, config_value) VALUES (1, 'database_version', '49'); `-->
                   <filterchain>
                     <linecontains>
-                      <contains value="'database_version'" />
+                      <contains value="'database_version'"/>
```

Comment:
We should update the pom formatter to mimic what maven release plugin does; I think I have this working for BRMO: https://github.com/B3Partners/brmo/blob/99fb85c91c32d4ff1aa0bde2b122d3def4c1fb6e/pom.xml#L1183-L1192

Reply:
Added these, seems to work!